tools/goctl/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/zeromicro/go-zero/tools/goctl
 
-go 1.20
+go 1.21
```

Comment:
Does this version have to be upgraded to 1.21? I recommend staying consistent with go-zero's version